src/components/toolbar.tsx

```diff
@@ -362,10 +364,21 @@ export class ToolbarComponent extends BaseComponent<IProps, IState> {
     }
   };
 
+  private getLoggerParameters = (otherParams: Record<string, any> = {}) => {
+    const { document, section } = this.props;
+    if (document) {
+      return {documentId: document.key, ...otherParams};
```

Comment:
This does not match the usual parameters logged about a document in `logDocumentEvent`, which include `documentKey` (not id), `documentTitle`, `documentHistoryId`, etc. I would call out to `logDocumentEvent` and `logCurriculumEvent` if possible.

Reply:
Fixed in https://github.com/concord-consortium/collaborative-learning/pull/2481/commits/b3693daa290edb8d1960d45d954739bd776337bf (and https://github.com/concord-consortium/collaborative-learning/pull/2481/commits/534c3428bfb685abe4c5ce3d98848dbc2754d210)